java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateCompactionJobs.java

```diff
@@ -109,9 +109,9 @@ private void createJobsForTable(TableProperties tableProperties, StateStore stat
         CompactionStrategy compactionStrategy = objectFactory
                 .getObject(tableProperties.get(COMPACTION_STRATEGY_CLASS), CompactionStrategy.class);
         LOGGER.debug("Created compaction strategy of class {}", tableProperties.get(COMPACTION_STRATEGY_CLASS));
-        compactionStrategy.init(instanceProperties, tableProperties);
+        compactionStrategy.init(instanceProperties, tableProperties, fileReferences, allPartitions);
 
-        List<CompactionJob> compactionJobs = compactionStrategy.createCompactionJobs(fileReferencesWithJobId, fileReferencesWithNoJobId, allPartitions);
```

Comment:
The lists that were passed in here now only seem to be used if we're forcing compaction of all files?